packages/0xcert-ethereum-utils-contracts/src/contracts/utils/bytes-to-types.sol

```diff
@@ -40,16 +42,15 @@ contract BytesToTypes
     assembly { x := mload(add(_input, _offst)) } // solhint-disable-line
     if (x == 0) {
       _output = false;
-    } else {
```

Comment:
Does this save any gas? Otherwise I would leave it as it for better readability.

Reply:
Fixed